src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Win32.cs

```diff
@@ -263,6 +283,22 @@ private bool CloseMainWindowCore()
             return true;
         }
 
+        /// <summary>Gets the caption of the main window of the process.</summary>
+        /// <value>The main window title of the process.</value>
+        /// <remarks>A process has a main window associated with it only if the process has a graphical interface. If the associated process does not have a main window (so that <see cref="System.Diagnostics.Process.MainWindowHandle" /> is zero), or if the system can't determine that there's a main window (such as may be the case on some Unix platforms), <see cref="System.Diagnostics.Process.MainWindowTitle" /> is an empty string ("").
+        /// If you have just started a process and want to use its main window title, consider using the <see cref="O:System.Diagnostics.Process.WaitForInputIdle" /> method to allow the process to finish starting, ensuring that the main window handle has been created. Otherwise, the system throws an exception.
+        /// <format type="text/markdown"><![CDATA[
+        /// > [!NOTE]
+        /// >  The main window is the window that currently has the focus; note that this might not be the primary window for the process. You must use the <xref:System.Diagnostics.Process.Refresh%2A> method to refresh the <xref:System.Diagnostics.Process> object to get the most up to date main window handle if it has changed.
```

Comment:
```suggestion
        /// > The main window is the window that currently has the focus. This might not be the primary window for the process. Use the <xref:System.Diagnostics.Process.Refresh%2A> method to refresh the <xref:System.Diagnostics.Process> object to get the most up-to-date main window handle if it has changed.
```